tests/kernel/bigquery_tests.py

```diff
@@ -49,15 +49,43 @@ class TestCases(unittest.TestCase):
   @mock.patch('google.datalab.Context.default')
   def test_udf_cell(self, mock_default_context, mock_notebook_environment):
     env = {}
+    mock_default_context.return_value = TestCases._create_context()
+    mock_notebook_environment.return_value = env
+
+    with self.assertRaises(Exception):
```

Comment:
Definitely better to have this assertRaises than no test, and probably not a big deal that it is not more specific, but I have seen cases with a test like this where a later code changes was causing some other exception to be thrown, and it was not caught by the unit test.

For the future, maybe you could use assertRaisesRegexp and include a piece of the expected error message.